tests/e2e/e2e.sh

```diff
@@ -18,12 +18,29 @@ set -eux
 
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
 ROOT=$(cd ../../../; pwd)
+HUB=istio-testing
```

Comment:
it is better to accept these from the `Makeile.core.mk` file directly rather than have to maintain them in two places.  On the call to `e2e.sh` i'd recommend HUB=$(HUB) TAG=$(TAG) tests/e2e/e2e.sh such that user overrides of HUB or TAG are used during the build process.